sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -109,7 +114,9 @@
 
     @Override
     public Map<String, PTransformRunnerFactory> getPTransformRunnerFactories() {
-      return ImmutableMap.of(ParDoTranslation.CUSTOM_JAVA_DO_FN_URN, new Factory());
+      return ImmutableMap.of(
+          PTransformTranslation.PAR_DO_TRANSFORM_URN, new NewFactory(),
```

Comment:
<!--new_thread; commit:5300be61ccac7a006ec7f2b8f3824cddb4dc75e8; resolved:0-->
Did you name this just for the delightful `new NewFactory()`?

Reply:
:). I just needed a new name for the new factory. The existing one will go away eventually.